recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "zfit-physics" %}
-{% set version = "0.8.1" %}
+{% set version = "0.9.0" %}
```

```suggestion
{% set version = "0.9.0" %}
{% set python_min = "3.10" %}
```

`python_min` defaults (globally) to `3.9` currently (conda-forge will bump it to `3.10` at some point in the future after Python `3.9` goes EOL), so set it in the recipe to `3.10` (given the change in https://github.com/zfit/zfit-physics/compare/0.8.1...0.9.0). I recommend that everyone set this manually all the time (c.f. the hints here https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python)